tap_redshift/__init__.py

```diff
@@ -170,13 +175,18 @@ def schema_for_column(c):
         result.type = 'string'
         result.format = 'date'
 
+    elif column_type in GEOMETRY:
+        LOGGER.info("in other types")
+        result.type = 'string'
+        result.format = 'symon.geo'
+
     else:
         result = Schema(None,
                         inclusion='unsupported',
                         description='Unsupported column type {}'
                         .format(column_type))
 
-    if column_nullable == 'yes':
+    if column_nullable == 'yes' and result.type != None:
```

Comment:
`is not None`

Reply:
also updated this so it sets ["null"] always